overlay/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+
```

Comment:
If we're not going to include the overlay module in the cmake build for now, perhaps we should just remove this file?